src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css

```diff
@@ -1379,6 +1379,11 @@ h1,h2,h3,h4,h5,h6{
 	transform: translate(-50%, -50%);
 	top: 50%;
 }
+
+.checkbox_field input {
+    margin: 10px 10px 0 0;
```

Comment:
@darek-grund could you check this style? #195

Reply:
Forgot to remove it. I've already done it.